Formula/uftp.rb

```diff
@@ -0,0 +1,45 @@
+class Uftp < Formula
+  desc "secure, reliable, efficient multicast file transfer program"
+  homepage "http://uftp-multicast.sourceforge.io/"
```

Comment:
fixed now.  I skimmed over the protocol portion of the URL.  It's unfortunate that sourceforge is 301'ing back to the http url - 301 is forever.